src/providers/azure.rs

```diff
@@ -631,23 +652,26 @@ mod azcmd {
                 "--generate-ssh-keys",
             ])
             .output()
-            .await?;
+            .await
+            .wrap_err("az vm create")?;
 
-        if !out.status.success() {
-            return Err(format_err!("Failed to create vm {}", name))
-                .context(String::from_utf8(out.stderr).unwrap())?;
-        }
+        eyre::ensure!(
+            out.status.success(),
+            "failed to create vm: {}",
+            String::from_utf8_lossy(&out.stderr)
```

Comment:
Now I want to add support for custom sections to `color-eyre` I think it would be nice if it could print a report like:

```
Error:
     0: ...

Stderr:
     ....

---- SPANTRACE ----
....
```

Reply:
Ooooh, yes, that'd be awesome! That way I could include both stderr, the exit status code, and other stuff!